src/commands.js

```diff
@@ -1,11 +1,54 @@
+import database from './database'
+import getHistory from './get_history'
+import remove from './remove'
+
+import debug from 'debug'
+var error = debug('ephembot:commands')
+var log = debug('ephembot:commands')
+
+// 15 minutes is the default
+const defaultLevel = 15
+
+// talon - Denver-Devs:
+const token = 'xoxp-4015048983-4025456865-4309485153-f20f9c'
```

Comment:
I just re-issued this token 'cause I'm not sure I should have committed it :see_no_evil: